includes/wc-stock-functions.php

```diff
@@ -395,9 +395,6 @@ function wc_release_stock_for_order( $order ) {
  * @return int
  */
 function wc_get_low_stock_amount( WC_Product $product ) {
-	if ( $product->is_type( 'variation' ) ) {
-		$product = wc_get_product( $product->get_parent_id() );
-	}
```

Comment:
This is potentially bw incompatible. I could create a new function, but I feel like it should not be a big issue. I scanned the marketplace for usage of this function and don't see any potential problems.